test/test_config.py

```diff
@@ -104,3 +104,13 @@ def test_ezclick_ezdrag():
     btn = config.EzDrag("A-2", cmd)
     assert btn.button == "Button2"
     assert btn.modifiers == [config.EzClick.modifier_keys["A"]]
+
+
+def test_screen_underbar_methods():
+    one = config.Screen(10, 10, 10, 10)
```

Comment:
These aren't positional arguments.
```suggestion
    one = config.Screen(x=10, y=10, width=10, height=10)
```

Reply:
Yeah, derp! Thank you!